presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/ElasticsearchConnectorConfig.java

```diff
@@ -105,13 +105,15 @@ public ElasticsearchConnectorConfig setScrollTimeout(Duration scrollTimeout)
         return this;
     }
 
+    @Deprecated
     @NotNull
     @Min(1)
     public int getMaxHits()
```

Comment:
`getMaxHits()` is now unused. I'd prefer to remove this config, adding this on top of the class:
```java
@DefunctConfig({
        "elasticsearch.max-hits",
})
```

If there is a strong reason not to remove this now, then let's have a TODO here and cleanup later (soonish).